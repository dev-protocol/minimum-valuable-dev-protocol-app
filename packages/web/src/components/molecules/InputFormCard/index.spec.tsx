import React from 'react'
import { render } from '@testing-library/react'
import { InputFormCard } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/fixtures/dev-kit/hooks')

describe(`${InputFormCard.name}`, () => {
  test('Snapshot', () => {
    const component = render(<InputFormCard label="Holder" onSubmitStake={() => {}} suffix="DEV" />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
