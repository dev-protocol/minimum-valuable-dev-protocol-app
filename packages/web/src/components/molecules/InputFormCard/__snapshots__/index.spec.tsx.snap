// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputFormCard Snapshot 1`] = `
<body>
  <div>
    <div
      class="ant-card ant-card-bordered"
      style="max-width: 961px;"
    >
      <div
        class="ant-card-body"
      >
        <div
          class="ant-row"
        >
          <div
            style="font-size: 24px; line-height: 32px; color: rgb(0, 0, 0); padding: 0px 0px 32px 0px;"
          >
            Holder
          </div>
        </div>
        <div
          class="ant-row"
        >
          <span
            class="ant-input-search ant-input-search-enter-button ant-input-search-large ant-input-group-wrapper ant-input-group-wrapper-lg"
            style="max-width: 578px;"
          >
            <span
              class="ant-input-wrapper ant-input-group"
            >
              <span
                class="ant-input-search ant-input-search-enter-button ant-input-search-large ant-input-affix-wrapper ant-input-affix-wrapper-lg"
              >
                <input
                  class="ant-input ant-input-lg"
                  type="text"
                  value=""
                />
                <span
                  class="ant-input-suffix"
                >
                  DEV
                </span>
              </span>
              <span
                class="ant-input-group-addon"
              >
                <button
                  class="ant-btn ant-input-search-button ant-btn-primary ant-btn-lg"
                  type="button"
                >
                  <span>
                    Stake
                  </span>
                </button>
              </span>
            </span>
          </span>
        </div>
      </div>
    </div>
  </div>
</body>
`;
